### DISCUSSION TITLE
How to follow the development of solid-start? (neither changelogs nor releases)

### DISCUSSION THREAD

**Discussion by Torsten85** [solidjs/solid-start#1248] [General]
Hello,
I really like solidjs and also solid-start, but it's very hard to follow the development.
After the release of beta 2, I saw multiple new verions in npm (0.4.0 - 0.4.8) but I don't find any changelog (the one in the repo is > 5 month old!) and there are no github releases.

What did change? What got fixed? I've no clue.

**Comment by thednp** [comment]
I usually check the [commits](https://github.com/solidjs/solid-start/commits/main/)

**Reply by Torsten85** [reply]
that would kind of work if there were tags for releases

**Comment by ryansolid** [comment]
We intend to do better here. There was a lot changing in early stages of 0.4.x. As stuff stabilizes I intend to get on semver. We haven't been there to date. But things are getting there now.

**Reply by ryansolid** [reply]
Changesets have been added.

**Reply by Torsten85** [reply]
great, thanks! Looking forward to use solid start in production!
